ztp/source-crs/PtpConfigForHA.yaml

```diff
@@ -0,0 +1,23 @@
+apiVersion: ptp.openshift.io/v1
```

Comment:
@aneeshkp There are other configs available in the source-crs for HA. Would you mind adding a comment at the beginning on when and why user needs to use it?